pyproject.toml

```diff
@@ -1,17 +1,17 @@
 [build-system]
-requires = ["setuptools", "versioneer[toml]==0.29"]
+requires = ["setuptools", "versioneer[toml]==0.29", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "sotodlib"
 readme = "README.md"
 description = "Simons Observatory TOD Simulation and Processing"
-license = {file = "LICENSE"}
+license-files = ["LICENSE"]
```

Comment:
Just wanted to point out that with this original syntax PyPI was trying to display the full license text in the sidebar, which I found strange. I ended up dropping this from packages I maintain in favor of the SPDX license expression (more on that in another comment). See for instance: https://github.com/simonsobs/ocs/pull/406 (and the corresponding example on PyPI: https://pypi.org/project/ocs/0.11.3rc1/)

That said, I'm not sure if that's still an issue or not. (Or whether you mind that.)